src/Middlewares/PermissionMiddleware.php

```diff
@@ -9,7 +9,13 @@ class PermissionMiddleware
 {
     public function handle($request, Closure $next, $permission)
     {
+        $routeNameRedirect = config('permission.unauthorized_route_name_redirect');
```

Comment:
Don't forget to update this to a relevant variable name for the updated config key `$unauthorizedRedirectUrl` or something.

Reply:
@AlexVanderbist, yes i will :)